docs/windows.md

```diff
@@ -254,7 +199,7 @@ Bcdedit.exe -set TESTSIGNING ON
 Restart-Computer
 ```
 
-Then, install the OVS using the script.
```

Comment:
Not introduced by this PR, but I saw both "Worker Node" and "worker Nodes" in the titles, maybe update them as well. Thanks.